pkg/trace/obfuscate/sql_test.go

```diff
@@ -542,6 +542,10 @@ ORDER BY [b].[Name]`,
 			`SELECT a :: VARCHAR(255) FROM foo WHERE foo.name = 'String'`,
 			`SELECT a :: VARCHAR ( ? ) FROM foo WHERE foo.name = ?`,
 		},
+		{
+			"/* { \"traceId\": \"6e9b5190-112a-487a-94ff-f33056a95ddc\", \"clientId\": \"35c1ccc0-a83c-4812-a189-895e9d4dd223\", \"context\": \"US\", \"mask\": \"c1d1abd4321677f94ff1754c80c860e34432e48a\", \"dataQueryId\": \"70004331c539496eb4fdcc7f86a553cc\", \"dataQueryName\": \"nas_min_becr_date_created\", \"modelIds\": [ \"f7b6f2250b014c2899545adccccae0d4\" ], \"entityIds\": [ \"a75f40d3b4b84e1690924811d3828ff7\" ], \"modelAuthorizations\": { \"f7b6f2250b014c2899545adccccae0d4\": { \"name\": \"nas_becr_export\", \"permissionLevelsConfigured\": [] } } } */ SELECT MIN(`scoped_49a39c4cc9ae4fdda07bcf49e99f8224`.`scoped_8720d2c0e0824ec2910ab9479085839c`) AS `MIN_BECR_DATE_CREATED` FROM (SELECT `49a39c4cc9ae4fdda07bcf49e99f8224`.`submittedOn` AS `scoped_8720d2c0e0824ec2910ab9479085839c`, `49a39c4cc9ae4fdda07bcf49e99f8224`.`domain` AS `scoped_847e4dcfa1c54d72aad6dbeb231c46de`, `49a39c4cc9ae4fdda07bcf49e99f8224`.`eventConsumer` AS `scoped_7b2f7b8da15646d1b75aa03901460eb2`, `49a39c4cc9ae4fdda07bcf49e99f8224`.`eventType` AS `scoped_77a1b9308b384a9391b69d24335ba058` FROM (`SorDesignTime`.`businessEventConsumerRegistry_947a74dad4b64be9847d67f466d26f5e` AS `49a39c4cc9ae4fdda07bcf49e99f8224`) WHERE (`49a39c4cc9ae4fdda07bcf49e99f8224`.`systemData.ClientID`) = ('35c1ccc0-a83c-4812-a189-895e9d4dd223')) AS `scoped_49a39c4cc9ae4fdda07bcf49e99f8224` WHERE ((`scoped_49a39c4cc9ae4fdda07bcf49e99f8224`.`scoped_847e4dcfa1c54d72aad6dbeb231c46de`) = ('Benefits') AND ((`scoped_49a39c4cc9ae4fdda07bcf49e99f8224`.`scoped_7b2f7b8da15646d1b75aa03901460eb2`) = ('benefits') AND (`scoped_49a39c4cc9ae4fdda07bcf49e99f8224`.`scoped_77a1b9308b384a9391b69d24335ba058`) = ('DMXSync'))); ",
```

Comment:
The comment adds a lot of noise, I'd suggest to remove it (comment stripping is covered in other tests).